app/controllers/api/suppliers_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Api
+  class SuppliersController < ApiController
+    def locations
+      paginate locations_from_supplier_moves
```

Comment:
Would be good to add support for includes here, these can be the same as the ones on the locations reference data endpoint I think. Something like:

```ruby
        paginate locations_from_supplier_moves, include: included_relationships
```

You'll also need to add this private method

```ruby
      def supported_relationships
        LocationSerializer::SUPPORTED_RELATIONSHIPS
      end
```

Reply:
Good point, it might be useful to FE.